README.md

```diff
@@ -63,7 +63,7 @@ First of all, clone the Git repository:
 Then run:
 
     $ mkdir _build && cd _build
-    $ meson ..
+    $ meson ..    # sometimes it's "meson.py"
```

Comment:
Meson only names the file `meson.py` on Windows, so it isn't a vague "sometimes" it is "on Windows".

And what I meant was Meson probably should treat msys2 special and *not* have the `.py` suffix for it. This is probably something that should be discussed upstream (or msys could even patch that).